src/index.js

```diff
@@ -0,0 +1,62 @@
+// import _ from 'lodash';
+import './style.css';
+
+const tasksList = [];
+let ids = 0;
+
+function Task(description, completed, index) {
+  this.description = description;
+  this.completed = completed;
+  this.index = index;
+}
```

Comment:
- [ ] Kindly change these and other regular functions to es6 arrow function syntax following the JavaScript best practices guide.